datanode/docker/Dockerfile_storageapi

```diff
@@ -32,19 +32,21 @@
 #   INTERUSS_API_PORT: Port on which the InterUSS API will be served (default 8121)
 #   INTERUSS_API_SERVER: IP address that will respond to InterUSS API requests (default 0.0.0.0)
 #   INTERUSS_VERBOSE: True for more log messages from the InterUSS API (default true)
+#   INTERUSS_TESTID: Force testing mode with test data located in specific test id
 
 FROM python:2
 
 WORKDIR /usr/src/app
 
-RUN pip install --no-cache-dir python-dateutil kazoo flask PyJWT djangorestframework cryptography pyopenssl
+RUN pip install --no-cache-dir python-dateutil kazoo flask PyJWT djangorestframework cryptography pyopenssl pytz gunicorn
```

Comment:
Can we get rid of flask? What about pyopenssl?
If we can get rid of them, don't forget to delete the import statements too.